src/runtime/hexagon_remote/halide_remote.cpp

```diff
@@ -326,12 +356,24 @@ int halide_hexagon_remote_power_hvx_off() {
     return 0;
 }
 
-int halide_hexagon_remote_get_symbol_v2(handle_t module_ptr, const char* name, int nameLen,
-                                        handle_t *sym_ptr) {
-    *sym_ptr = reinterpret_cast<handle_t>(obj_dlsym(reinterpret_cast<elf_t*>(module_ptr), name));
+int halide_hexagon_remote_get_symbol_v3(handle_t module_ptr, const char* name, int nameLen, int usedl, handle_t *sym_ptr) {
+    if (usedl)
+       *sym_ptr = reinterpret_cast<handle_t>(dlsym(reinterpret_cast<elf_t*>(module_ptr), name));
+    else
+        *sym_ptr= reinterpret_cast<handle_t>(obj_dlsym(reinterpret_cast<elf_t*>(module_ptr), name));
     return *sym_ptr != 0 ? 0 : -1;
 }
 
+int halide_hexagon_remote_get_symbol_v2(handle_t module_ptr, const char* name, int nameLen, handle_t *sym_ptr) {
```

Comment:
This should be removed, it's redundant with the shim in halide_host.cpp and it's not part of the IDL.